src/discord.ts

```diff
@@ -86,6 +86,63 @@ discord.on(
   },
 );
 
+discord.on(
```

Comment:
Does this mean that any time someone on the list enters voice chat, they get the live streaming role? Does this include things like co-working and private voice channels on the server?

Reply:
Good catch there @SociableSteve! We can implement a ignore list for channels.
All I need is a list with channel ids to add to the configuration file.

Reply:
> Does this mean that any time someone on the list enters voice chat, they get the live streaming role?

No. They only get a role when they start streaming inside a voice channel.

Reply:
> Does this include things like co-working and private voice channels on the server?

I am not aware of any private voice channels on The Claw but I just realized that p4nth3rb0t is a moderator (or administrator?) so the bot can sniff those secret events too. It would be wise to include a filter to exclude those voice channels by ID.

I was thinking of something like:
```typescript
if (config.discord.liveStreamingIgnoredChannels.includes(oldVoiceState.channelID as string)) {
  return;
}
```

`config.discord.liveStreamingIgnoredChannels` would be something like a `string[]`. Would that be a solution to ignore voice channels or do you have a better suggestion?

Reply:
@SociableSteve I'd love to get a reply on my question, if possible. Thank you! 🙌🏻